app/lib/frontend/templates/views/pkg/info_box.dart

```diff
@@ -140,12 +142,14 @@ d.Node _linkAndBr(InfoBoxLink link) {
 }
 
 d.Node? _licenseNode({
-  required LicenseFile licenseFile,
+  required List<License> licenses,
   required String licenseUrl,
 }) {
+  final paths = licenses.map((e) => e.path).toSet().toList();
+  final ids = licenses.map((e) => e.spdxIdentifier).toSet().toList();
   return d.fragment([
-    d.text('${licenseFile.shortFormatted} ('),
-    d.a(href: licenseUrl, text: licenseFile.path),
+    d.text('${ids.join(', ')} ('),
+    d.a(href: licenseUrl, text: paths.join(', ')),
```

Comment:
There is only going to be one path here, right? unless we change the detection code... but then won't the URL be wrong?

Reply:
Unfortunately no, this is an issue. I've commented about this in the attached issue: we need to extract and store the file content via `pana`, otherwise we have a divergence (regardless of detecting one or multiple licenses), as the package reader code extracts only the first candidate and doesn't have any analysis in it. Or, alternatively, we should run the analysis at upload time, and re-run it periodically.

Reply:
okay, that makes sense...